lib/options-manager.js

```diff
@@ -179,9 +179,9 @@ const mergeWithFileConfigs = async (files, options) => {
 };
 
 /**
- * Generate a unique and consitent path for the temporary `tsconfig.json`.
```

Comment:
Just out of curiosity, what's the reason for changing the function comment format?
[The JSDoc](https://jsdoc.app/about-getting-started.html) use the format with each intermediate line starting with a `*`.

Reply:
btw I didn't noticed you have changed the other comments, otherwise I would used your preferred format here.

Reply:
I think it's just needless noise and it makes it harder to edit things, especially annoying to add multi-line code blocks in the comment.

Reply:
Make sense. My IDE format it with the `*` automatically and it when I insert a new line so I never really paid much attention.
I'll use your preferred format in future PRs!

Reply:
Don't worry. I didn't want to bother you with nitpick style issues. I'm just very particular about that. Hence why I made XO in the first place :P